itkwidgets/viewer.py

```diff
@@ -124,6 +127,9 @@ def update_screenshot(self, base64_image):
                 </script>
             ''')
         self.img.display(html)
+    
+    def save_roi(self, roi):
+        print('save roi', roi)
```

Comment:
I'm not seeing this actualy print, so missing something...

Reply:
Print statements do not bubble up correctly from the ViewerRPC class. You can use `api.log(f'save roi {roi}')` instead and the message will be output in the console log

Reply:
Ah, thank you.  This is the output: 
```
Plugin 5b4de75c-9cb8-490b-8c32-28c9ae68f0c4: save roi {'name': 'Image', 'layerName': 'Image', 'croppingPlanes': [{'origin': [-0.5, -0.5, -0.5], 'normal': [1, 0, 0]}, {'origin': [127.5, 127.5, 255.5], 'normal': [-1, 0, 0]}, {'origin': [-0.5, -0.5, -0.5], 'normal': [0, 1, 0]}, {'origin': [127.5, 127.5, 255.5], 'normal': [0, -1, 0]}, {'origin': [-0.5, -0.5, -0.5], 'normal': [0, 0, 1]}, {'origin': [127.5, 127.5, 255.5], 'normal': [0, 0, -1]}]}
```